src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -151,15 +186,21 @@ export class CreateEditEventsComponent implements OnInit {
     }
     const tagsArr: Array<string> = this.tags.filter((tag) => tag.isActive).reduce((ac, cur) => [...ac, cur.nameEn], []);
 
-    const sendEventDto: EventDTO = {
-      title: this.titleForm.value,
-      description: this.editorHTML,
+    let sendEventDto: EventDTO = {
+      title: this.eventFormGroup.get('titleForm').value,
+      description: this.eventFormGroup.get('description').value,
       open: this.isOpen,
       datesLocations: datesDto,
       tags: tagsArr
     };
-
-    if (this.checkdates && this.titleForm.valid && this.contentValid) {
+    if (this.editMode) {
+      sendEventDto = {
+        ...sendEventDto,
+        imagesTodelete: this.imagesToDelete
+      };
+    }
+    const test = true;
+    if (this.checkdates && this.eventFormGroup.valid && tagsArr.length && test) {
```

Comment:
Why you use test here? It is always true

Reply:
@hnativlyubomyr Hi, I got some issues with Github, this change is not related to this pr.